helm/ida-otp/values.yaml

```diff
@@ -302,8 +314,8 @@ volumePermissions:
   enabled: false
   image:
     registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: "10"
+    repository: mosipid/os-shell
+    tag: "12-debian-12-r46"
     pullPolicy: Always
     ## Optionally specify an array of imagePullSecrets.
     ## Secrets must be manually created in the namespace.
```

Comment:
repository: mosipid/authentication-otp-service
  tag: 1.3.0-beta.2 change to    repository: mosipqa/authentication-otp-service
  tag: 1.3.x